prov/util/src/util_mem_monitor.c

```diff
@@ -78,10 +79,53 @@ static size_t ofi_default_cache_size(void)
 	return cache_size;
 }
 
+static int ofi_monitors_update(struct ofi_mem_monitor **monitors)
+{
+	int ret = 0;
+	enum fi_hmem_iface iface;
+	struct ofi_mem_monitor *monitor;
+
+	if (!monitors) {
+		return -FI_ENOSYS;
+	}
+
+	pthread_mutex_lock(&mm_state_lock);
+	for (iface = 0; iface < OFI_HMEM_MAX; iface++) {
+		monitor = monitors[iface];
+		if (monitor == NULL)
+			continue;
+
+		switch (monitor->state) {
+			case FI_MM_STATE_STARTING:
+				ret = monitor->start(monitor);
+				if (ret) {
+					monitor->state = FI_MM_STATE_IDLE;
+					FI_WARN(&core_prov, FI_LOG_MR,
+						"Failed to start %s memory monitor: %s\n",
+						fi_tostr(&iface, FI_TYPE_HMEM_IFACE), fi_strerror(-ret));
+
+					goto out;
+				}
+				monitor->state = FI_MM_STATE_RUNNING;
+				break;
+			case FI_MM_STATE_STOPPING:
+				monitor->stop(monitor);
+				monitor->state = FI_MM_STATE_IDLE;
+				break;
+			default:
+				break;
```

Comment:
please outdent the case lines to align with the switch.

Reply:
Done